rust/main/config/mainnet_config.json

```diff
@@ -7695,6 +7695,67 @@
       "index": {
         "from": 12446115
       }
+    },
+    "trumpchain": {
+      "blockExplorers": [
+        {
+          "apiUrl": "https://explorer.trumpchain.dev/api",
+          "family": "blockscout",
+          "name": "TRUMPCHAIN Explorer",
+          "url": "https://explorer.trumpchain.dev"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "estimateBlockTime": 1,
+        "reorgPeriod": 0
```

Comment:
are we leaving this 0?

Reply:
yes, since it's an arb nitro chain with low usage (comes from the registry PR)